filters/src/main/java/com/matecat/converter/server/MatecatConverterServer.java

```diff
@@ -18,51 +14,37 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
-import java.net.BindException;
-import java.net.InetAddress;
-import java.net.URL;
+import java.net.*;
 
 
 /**
  * Matecat converter server
  */
 public class MatecatConverterServer {
 
-    // Logger
     private static final Logger LOGGER = LoggerFactory.getLogger(MatecatConverterServer.class);
-
-    // Used port
-    private int serverPort;
-
-    // Server
     private Server server;
-    private String localIP, externalIP;
+    private final int serverPort;
+    private static String localIP, externalIP;
 
 
     /**
      * Constructor which will use the default port
      */
     public MatecatConverterServer() {
-        try {
-            int port = Config.serverPort;
-            if (port <= 0)
-                throw new Exception();
-            this.serverPort = port;
-            init();
-        }
-        catch (Exception e) {
-            throw new RuntimeException("There is no default port specified in the configuration");
-        }
+        this.serverPort = Config.serverPort;
+        init();
     }
 
 
     /**
      * Constructor admitting a configured port
+     *
      * @param serverPort Port to use
      */
     public MatecatConverterServer(int serverPort) {
         if (serverPort < 0)
```

Comment:
Same here.

Reply:
I will see what I could do but please note that this constructor is only used in tests.